src/components/controls/mouse-controls.js

```diff
@@ -0,0 +1,157 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+var debug = require('../../utils/debug');
+
+var warn = debug('components:mouse-controls:warn');
+
+/**
+ * Mouse-controls component.
+ *
+ * Updates rotation with mousedrag or pointerlock input.
+ */
+module.exports.Component = registerControls('mouse-controls', {
+  schema: {
+    enabled: { default: true },
+    pointerlockEnabled: { default: false },
```

Comment:
Hehe, well it isn't removing code. We're just talking about not adding code.

@donmccurdy Let's publish PL outside core. Once we have lazy-loading components, it will be a better story for consumption.